bucket/ddc_bucket/cdn_node/messages.rs

```diff
@@ -16,10 +16,10 @@ impl DdcBucket {
         self.impl_grant_permission(manager, permission, is_trusted)
     }
 
-    pub fn message_cdn_node_create(&mut self, node_params: Params) -> Result<NodeId> {
+    pub fn message_cdn_node_create(&mut self, node_params: Params, pubkey: AccountId) -> Result<NodeId> {
         let provider_id = Self::env().caller();
 
-        let node_id = self.cdn_nodes.create(provider_id, 0);
+        let node_id = self.cdn_nodes.create(provider_id, 0, pubkey).unwrap();
```

Comment:
I would rename all occurrences of `pubkey` to `pub_key` according to conventions